en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/rate-limiting/distributed-throttling.md

```diff
@@ -1,79 +1,79 @@
-      # Data publisher object pool configurations
-      [enforcer.throttling.publisher.pool]
-        maxIdleDataPublishingAgents = 1000
-        initIdleObjectDataPublishingAgents = 200
-        # Data publisher thread pool configurations
-        publisherThreadPoolCoreSize = 200
-        publisherThreadPoolMaximumSize = 1000
-        publisherThreadPoolKeepAliveTime = 200
-      [enforcer.throttling.publisher.agent]
-        # SSL Protocols
-        sslEnabledProtocols = "TLSv1.2"
-        # ciphers
-        ciphers="TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256, TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA256, TLS_ECDH_RSA_WITH_AES_128_CBC_SHA256,TLS_DHE_RSA_WITH_AES_128_CBC_SHA256,TLS_DHE_DSS_WITH_AES_128_CBC_SHA256, TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA, TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDH_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_RSA_WITH_AES_128_CBC_SHA, TLS_DHE_DSS_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256  ,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256, TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_DSS_WITH_AES_128_GCM_SHA256  ,TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA,TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,SSL_RSA_WITH_3DES_EDE_CBC_SHA, TLS_ECDH_ECDSA_WITH_3DES_EDE_CBC_SHA,TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA,SSL_DHE_RSA_WITH_3DES_EDE_CBC_SHA, SSL_DHE_DSS_WITH_3DES_EDE_CBC_SHA,TLS_EMPTY_RENEGOTIATION_INFO_SCSV"
-        # The size of the queue event disruptor which handles events before they are published.
-        # The value specified should always be the result of an exponent with 2 as the base.
-        queueSize = 32768
-        # The maximum number of events in a batch sent to the queue event disruptor at a given time
-        batchSize = 200
-        # The number of threads that will be reserved to handle events at the time you start
-        corePoolSize = 1
-        # Socket timeout
-        socketTimeoutMS = 30000
-        # The maximum number of threads that should be reserved at any given time to handle events
-        maxPoolSize = 1
-        # The amount of time which threads in excess of the core pool size may remain idle before being terminated.
-        keepAliveTimeInPool = 20
-        # The time interval between reconnection
-        reconnectionInterval = 30
-        # TCP connection pool configurations (for data publishing)
-        maxTransportPoolSize = 250
-        maxIdleConnections = 250
-        evictionTimePeriod = 5500
-        minIdleTimeInPool = 5000
-        # SSL connection pool configurations (for authentication)
-        secureMaxTransportPoolSize = 250
-        secureMaxIdleConnections = 250
-        secureEvictionTimePeriod = 5500
-        secureMinIdleTimeInPool = 5000
+# Throttling configurations
```

Comment:
@pubudu538 , please check this config. I had a merge conflict on this.
![Mergely-Diff-online-merge-documents](https://user-images.githubusercontent.com/5195851/142986947-a184ea57-e3db-4dc2-b9c2-eca19d349eee.png)